lib/shoulda/matchers/active_record/have_db_index_matcher.rb

```diff
@@ -65,6 +65,56 @@ module ActiveRecord
       #
       # @return [HaveDbIndexMatcher]
       #
+      # has an index and an array of columns
+      #
+      #     class CreateBlogs < ActiveRecord::Migration
+      #       def change
+      #         create_table :blogs do |t|
+      #           t.integer :user_id
+      #           t.string :name
+      #         end
+      #
+      #         add_index :blogs, :user_id, :name
```

Comment:
Is this actually valid? I thought the syntax had to use an array, like this:

```ruby
add_index :blogs, [:user_id, :name]
```

Reply:
Not necessarily, but is valid.